packages/js-sdk/src/index.ts

```diff
@@ -9,7 +9,7 @@ const consoleLogPrefix = '@growthflags/js-sdk error:';
 
 export interface Constructor {
   publicApiKey: string;
-  env: string;
+  env: 'development' | 'staging' | 'production';
```

Comment:
Please, return back string. We already have 'demo', might have more in the future. Our API validates allowed types